crates/aptos/src/move_tool/mod.rs

```diff
@@ -320,6 +326,10 @@ pub struct TestPackage {
     #[clap(long, short)]
     pub filter: Option<String>,
 
+    /// A boolean value to skip warnings.
+    #[clap(long, short = 'w')]
```

Comment:
Only feedback I'd have is we've been avoiding shorts because of confusion between inputs (though, I see right above it is a short, so I guess).  It's only this command so I'm fine with it.